extension/manifest.json

```diff
@@ -251,6 +251,12 @@
 			"js": ["scripts/features/bazaar-worth/ttBazaarWorth.entry.js"],
 			"run_at": "document_start"
 		},
+		{
+			"matches": ["https://www.torn.com/bazaar.php*"],
+			"css": ["scripts/features/bazaar-sub-vendor-items/ttSubVendorPriceItems.css"],
+			"js": ["scripts/features/bazaar-sub-vendor-items/ttSubVendorPriceItems.js"],

```

Comment:
If this really needs to run at document_start, make the extension `.entry.js` instead to make it clear based on the file name.